### PR TITLE
List the commands when new projects are created

### PR DESCRIPTION
**Pull request by rafaelcascalho**
## What do these changes do?
It shows a list of the commands of the project when a new project is created. The commands were mapped from the #93 to keep consistency, but I wrote shorter explanations in some cases thinking about newcomers. I didn't wrote unit tests for the functions I've created cause they pretty much show the results in the screen, but I tested it to make sure it works fine everytime the command is called.

## Are there changes in behavior for the user?
Yes! Now after he creates a new project a list of the possible commands is showed in the terminal.

## Related issue number
#103 

## Checklist

- [x] I think the code is well written
- [ ] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `103.feature` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: `Fix issue with non-ascii contents in doctest text files.`
[
Screenshot from 2019-09-13 17-54-31
](url)

### CODE REVIEW

**Comment by Arfey** [comment]
Thx for your contribution 👍

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -17,3 +17,5 @@ Oleh Kuchuk
 Sviatoslav Sydorenko
 Vasyl Navrotsky
 Vladyslav Ovchynnykov
+Alexey Dyatlov
```
Comment (Arfey) [inline CONTRIBUTORS.txt]: this line repeats twice `Alexey Dyatlov`

File: create_aio_app/main.py [hunk 2]
```diff
@@ -15,6 +16,22 @@
...
+
+def show_commands():
+    echo(click.style('\n    Common', fg='cyan'))
+    print_commands(COMMON)
+    echo(click.style('\n    Linters', fg='cyan'))
+    print_commands(LINTERS)
+    echo(click.style('\n    Database', fg='cyan'))
+    print_commands(DATABASE)
+
+
```
Comment (Arfey) [inline create_aio_app/main.py]: I think that we have better way to do this. We actually have feature which show all commands from `makefile`.

```
make help
```

two reasons to use that:

- don't repeat yourself
- auto generation of output. `make help` generate automation output and we don't need update this each time when we change `makefile`.

So, I propose use some like this:

```python
import os

try:
    os.system('make help')
except Exception:
    ...
```
We need to wrap this to `try` block because not each `os` has `make`. 

what do u think?

File: README.md [hunk 3]
```diff
@@ -28,14 +29,39 @@ or if u wanna use manual mode, enter only command below
...
+black - formats the code with black
+mypy - checks types with mypy
+lint - formats the code with flake8
+
+    Database
+
+psql - run the application
+migrations - stop the application
+migrate - run tests
+adev - shows help
```
Comment (Arfey) [inline README.md]: we have this description in https://create-aio-app.readthedocs.io/pages/commands.html. do u think that  it'll help to users? Also, this output user will see after created app in console.

**Comment by Arfey** [comment]
HI 🖐
any updates?

File: CONTRIBUTORS.txt (hunk 1 above)
Comment (rafaelcascalho) [reply CONTRIBUTORS.txt]: Oh, I've missed that.

File: create_aio_app/main.py (hunk 2 above)
Comment (rafaelcascalho) [reply create_aio_app/main.py]: I agree it's a better solution 👍

**Comment by rafaelcascalho** [comment]
Hey @Arfey 😄! Sorry the delay 
I'll take a look in the `commands` docs showed by your last comment tonight, and I'll reply and see if I can change it still this week
Sounds good?

**Comment by Arfey** [comment]
sure I'll wait

**Comment by rafaelcascalho** [comment]
Hey @Arfey , sorry to take so long.
I agree with what you said, and I'll try to make the changes today and push them.

File: README.md (hunk 3 above)
Comment (rafaelcascalho) [reply README.md]: I agreed that it could not be necessary, so, instead, I just added a link into the `make commands docs`, even though it might seem repetitive, since at the bottom there's already a link to the `docs`, I think it'd help the users to a quicker start. What do u think?

**Comment by Arfey** [comment]
Problem with ci:

```
bandit -r ./create_aio_app -x create_aio_app/template -s B101
756[main]	INFO	profile include tests: None
757[main]	INFO	profile exclude tests: None
758[main]	INFO	cli include tests: None
759[main]	INFO	cli exclude tests: B101
760[main]	INFO	running on Python 3.6.7
761Run started:2020-03-16 03:05:58.211149
762
763Test results:
764>> Issue: [B605:start_process_with_a_shell] Starting a process with a shell, possible injection detected, security issue.
765   Severity: High   Confidence: High
766   Location: ./create_aio_app/main.py:23
767   More Info: https://bandit.readthedocs.io/en/latest/plugins/b605_start_process_with_a_shell.html
76822	        os.chdir(f'{folder}/')
76923	        os.system(f'make help')
77024	    except expression as identifier:
```

add `# nosec` for this line for skip that

**Comment by rafaelcascalho** [comment]
Hey @Arfey !
Thanks for the patience.
So, I've corrected using the `# nosec` in the failing lines but it's still not building correctly. I've checked the [ci build log](https://travis-ci.com/github/aio-libs/create-aio-app/jobs/318897483) and could not figure it out.
These are the log last lines
```
1382 test_1_f94b1c5b4b11 | ============================== 2 passed in 0.63s ===============================
1383 Stopping project_new_test     ... 
1384 Stopping project_new_postgres ... 
1385 The command "make test" exited with 0.
1386 store build cache
1401 
1402
1403 Done. Your build exited with 1.
```
I've found this error in the ci, but I'm not sure if that's the problem
```
802 flake8 create_aio_app --exclude create_aio_app/template
803 create_aio_app/main.py:7:80: E501 line too long (81 > 79 characters)
804 create_aio_app/main.py:21:5: F841 local variable 'identifier' is assigned to but never used
805 create_aio_app/main.py:21:12: F821 undefined name 'expression'
806 create_aio_app/main.py:47:80: E501 line too long (88 > 79 characters)
807 Makefile:14: recipe for target 'flake' failed
808 make: *** [flake] Error 1
809 The command "make flake" exited with 2.
```
Am I missing something?

**Comment by Arfey** [comment]
some more... 😀

I think that u spend a lot of time to check error in CI. U can reproduce this issue local. Just put

```
make test
```

if after this command u'll not have a errors

```
echo $?  // return 0
```
it's mean that all is fine.

Error which u see is error connected with flake8. U need prettify ur code in `main.py`. if u'll have so type me and i'll try to help.

**Comment by rafaelcascalho** [comment]
Hey @Arfey !

I've corrected the `flake8` linting issue, you can check the `main.py` file but basically, I've modified a one too long import, and refactored the `directory already exists` echo message.

It's also still showing that requires an old change in the old version of the `show_commands` while I still was printing many commands by hand instead of using `make help`, as of right now.

So, that's it I guess :smile:

File: create_aio_app/main.py [hunk 4]
```diff
@@ -15,38 +15,47 @@
...
         return
 
     folder = Path(result).name
 
-    echo(click.style(
-        '\n\nSuccessfully generated!\n',
-        fg='bright_green',
-    ))
-    echo('cd ' + click.style(f'{folder}/', fg='bright_blue'))
-    echo('make run\n\n')
```
Comment (Arfey) [inline create_aio_app/main.py:52]: a'u sure that this line is important here?

Comment (rafaelcascalho) [reply create_aio_app/main.py:52]: I'm not sure if the line was the same one, cause it seems that the main file you're referencing is different from the main I've modified. Not sure if that's a problem.

**Comment by Arfey** [comment]
These was long 7 months but u did it. I congrats u and thx 👍
